pkg/validation-webhook/snapshot.go

```diff
@@ -20,7 +20,7 @@ import (
 	"fmt"
 	"reflect"
 
-	volumesnapshotv1beta1 "github.com/kubernetes-csi/external-snapshotter/client/v3/apis/volumesnapshot/v1beta1"
+	volumesnapshotv1 "github.com/kubernetes-csi/external-snapshotter/client/v3/apis/volumesnapshot/v1beta1"
```

Comment:
this is odd? we call it v1 but it's actually v1beta1.

Reply:
ok, I"ll change that.

Reply:
Done